spec/features/javascript/search_config_admin_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+feature 'Search Configuration Administration', js: true do
+  let(:exhibit) { FactoryGirl.create(:default_exhibit) }
+  let(:exhibit_curator) { FactoryGirl.create(:exhibit_curator, exhibit: exhibit) }
+  before { login_as exhibit_curator }
+
+  describe "facets" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.